plugins/modem-manager/fu-sahara-loader.c

```diff
@@ -105,9 +107,18 @@ fu_sahara_loader_open(FuSaharaLoader *self, FuUsbDevice *usb_device, GError **er
 gboolean
 fu_sahara_loader_close(FuSaharaLoader *self, GError **error)
 {
+	if (!self->interface_found) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOTHING_TO_DO,
+				    "usb device interface was not found");
+		return FALSE;
+	}
 	if (!fu_device_close(FU_DEVICE(self->usb_device), error))
```

Comment:
I'm not sure what you're trying to protect against -- is `self->usb_device` invalid, or `error` invalid?

Reply:
As already stated in the other [PR](https://github.com/fwupd/fwupd/pull/8527#issuecomment-2694059770), I will fix this here. I will update this PR.

Reply:
I am now verify if the `self->usb_device` pointer is NULL.